modelling/src/test/java/org/axonframework/modelling/command/inspection/AnnotatedAggregateMetaModelFactoryTest.java

```diff
@@ -247,6 +267,26 @@ void testFindIdentifier() {
         assertEquals("id", inspector.routingKey());
     }
 
+    @Test
+    void testFindGetterIdentifier() {
+        AggregateModel<SomeGetterIdAnnotatedHandlers> inspector =
+                AnnotatedAggregateMetaModelFactory.inspectAggregate(SomeGetterIdAnnotatedHandlers.class);
+
+        assertEquals("SomeGetterIdAnnotatedHandlers", inspector.type());
+        assertEquals("id", inspector.getIdentifier(new SomeGetterIdAnnotatedHandlers()));
+        assertEquals("getId", inspector.routingKey());
+    }
+
+    @Test
+    void testFindMethodIdentifier() {
+        AggregateModel<SomeMethodIdAnnotatedHandlers> inspector =
+                AnnotatedAggregateMetaModelFactory.inspectAggregate(SomeMethodIdAnnotatedHandlers.class);
+
+        assertEquals("SomeMethodIdAnnotatedHandlers", inspector.type());
+        assertEquals("id", inspector.getIdentifier(new SomeMethodIdAnnotatedHandlers()));
+        assertEquals("calculatedId", inspector.routingKey());
```

Comment:
This scenario would be a valid use case where the `routingKey` should be set on the `@EntityId`/`@AggregateIdentifier`. Would be smart to be specific about this in the javadoc too.

Reply:
Javadoc for the `@EntityId` and `@AggregateIdentifier` has been adjusted to clarify this.